### PR TITLE
Feedback for the dog

### LINKED ISSUE #59: It is hard for FPS player to find dogs
Since the dogs are small it can be hard to see the dogs in FPS game mode. We need the human to have some kind of "cat sense" to show them where the dogs are.

### LINKED ISSUE #63: Need visual feedback on the status of dogs.
Are they fleeing, heavily damaged or attacking?

### PR DESCRIPTION
**Pull request by SorraTheOrc**
Add an icon to show the position of dogs in this block Closes #59 
Use vertex colors to show the status of the animal Closes #63

### CODE REVIEW

File: Packages/manifest.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "com.unity.ugui": "1.0.0",
     "com.unity.visualscripting": "1.7.5",
     "org.wizardscode.devlogger": "git@github.com:TheWizardsCode/DevLogger.git",
+    "wizardscode.character": "file:E:/WizardsCode/Character-Release",
```
Comment (bsimser) [inline Packages/manifest.json:16]: These need to be git references not file locations on your hard drive ;)

File: Packages/packages-lock.json [hunk 2]
```diff
@@ -159,6 +159,12 @@
       },
       "hash": "c4d65ec4b8ad13848e89deae6e41c17d9b2577a6"
     },
+    "wizardscode.character": {
+      "version": "file:E:/WizardsCode/Character-Release",
```
Comment (bsimser) [inline Packages/packages-lock.json:163]: Same thing here with the package reference doesn't work in other systems when it's E:/WizardsCode

**Review by bsimser — COMMENTED** [review]
If you can make the file reference changes point to github instead I'll merge this otherwise looks good to go.

**Comment by SorraTheOrc** [comment]
That is not necessary. I thought I had removed it. I will do so.
________________________________

**Pull request merged**